pkg/cmd/roachtest/test_runner.go

```diff
@@ -544,6 +544,9 @@ func (r *testRunner) runWorker(
 	// When this method returns we'll destroy the cluster we had at the time.
 	// Note that, if debug was set, c has been set to nil.
 	defer func() {
+		//TODO: Consider removing the test_run_id label here.
```

Comment:
TODO is missing username; it might be better to move it before the call to `c.Save` since presumably that's where `destroyState` would be extended to remove the labels.